hugegraph-llm/src/hugegraph_llm/demo/rag_demo/other_block.py

```
@@ -42,14 +45,47 @@ def create_other_block():
         out = gr.Textbox(label="Backup Graph Manually (Auto backup at 1:00 AM everyday)", show_copy_button=True)
     btn = gr.Button("Backup Graph Data")
     btn.click(fn=backup_data, inputs=inp, outputs=out)  # pylint: disable=no-member
+    # auto test llm
+    with gr.Accordion("Evaluation Model Settings (only support openai)", open=True):
+        with gr.Row():
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)